starter-code/src/functions-and-arrays.js

```diff
@@ -1,16 +1,83 @@
 // Iteration #1: Find the maximum
 
+const maxOfTwoNumbers = function (num1, num2) {
+  // if num1 is bigger than num2, return num1
+  if (num1 > num2) {
+    return num1
+    // if num2 is bigger than num1, return num2
+  } else if (num2 > 1) {
+    return num2
+  } else {
+    // if they are equal, return error message
+    console.log('Your numbers are equal.')
+  }
+}
+
+
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+let findLongestWord = function (array) {
+  var longest = "";
+  // if there is only one item in the array, return that item
+  if (array.length === 1) {
```

Comment:
I feel this 'if' statement is unnecessary. Even with a single element in your array, your 'else' statement would be handle it so you don't really need this one.